app/templates/drupal/drupal/8.x/Gruntconfig.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Would be better if we loaded the Gruntconfig as a JSON object and simply added/removed properties as needed.